### PR TITLE
CLOUDP-119550: Fixed incorrect tag for the manager

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

Resolves https://jira.mongodb.org/browse/CLOUDP-119550 for this fork

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [x] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).

### CODE REVIEW

**Comment by jianrongzhang89** [comment]
Hi @igor-karpukhin can you resubmit this PR against RHEcosystemAppEng/dbaas-refactor-rebased branch? This is where the RHODA release build is based on. Thanks.

**Comment by tchughesiv** [comment]
@igor-karpukhin @jianrongzhang89 changed base branch.

**Review by tchughesiv — CHANGES_REQUESTED** [review]
@igor-karpukhin please resolve conflict

File: Makefile [hunk 1]
```diff
@@ -11,6 +11,7 @@ VERSION ?= 0.8.0
 ifndef PRODUCT_VERSION
 PRODUCT_VERSION := $(shell git describe --tags --dirty --broken)
 endif
+
```
Comment (tchughesiv) [inline Makefile:14]: @igor-karpukhin after resolving conflicts ... is this PR still necessary if all we're adding is a new line?

Comment (igor-karpukhin) [reply Makefile:14]: Looks like this fix was in another PR as well. I'm closing this PR.

**Pull request closed without merging**